Release/6.16.0 (#1653)
onfido/onfido-sdk-ui: CHANGELOG.md

## [next-version]

## [6.16.0] - 2021-12-14

### Added

- Public: Added support for Dutch `nl_NL`.

### Changed

> - Public: Added `autoFocusOnInitialScreenTitle` SDK configuration option for integrators to override SDK auto focusing on the SDK's initial screen title on loading. This default behaviour might not be desirable for some host apps or sites as this could cause the browser to focus on the SDK when there is content or form inputs outside of the SDK that the end user should see, fill in first.

Reviewer: ```suggestion
- Public: Added `autoFocusOnInitialScreenTitle` SDK configuration option for integrators to override the SDK auto focusing on the initial screen's title on loading. The default behaviour may not be desirable for some host apps or sites as it could cause the browser to focus on the SDK, rather than content or form inputs outside of the SDK that the end user should see and fill in first.
```

- Upgrade `react-phone-number-input` to v3.1.38

> - Revert change which returns document type as 'unknown' in `onComplete` callback payload if Residence Permit selected. API now supports that document type for document uploads.

Reviewer: ```suggestion
- Revert change which returns document type as 'unknown' in `onComplete` callback payload if Residence Permit is selected. The API now supports Residence Permit as a document type for document uploads.
```
